docs/running-node.md

```diff
@@ -1,346 +1,59 @@
 # Running a DCLedger Node
 
```

Comment:
Probably we need to copy the paragraph from README with a description about node types (what is VN, ON, Sentry, etc.)